docs/API.md

```diff
@@ -18,12 +18,16 @@ There are a number of methods that a DB storage backend should implement:
     * .createSessionToken(tokenId, sessionToken)
     * .updateSessionToken(tokenId, sessionToken)
     * .sessionToken(id)
+    * .sessionTokenVerified(tokenId)
```

Comment:
For example, it's not obvious to me when we'd ever call `keyFetchToken` in preference to `keyFetchTokenVerified`